lms/djangoapps/instructor_task/tests/test_api.py

```diff
@@ -115,11 +115,12 @@ def _test_submit_with_long_url(self, task_function, student=None):
         problem_url_name = 'x' * 255
         self.define_option_problem(problem_url_name)
         location = InstructorTaskModuleTestCase.problem_location(problem_url_name)
-        with self.assertRaises(ValueError):
-            if student is not None:
-                task_function(self.create_task_request(self.instructor), location, student)
-            else:
-                task_function(self.create_task_request(self.instructor), location)
+        # creating the task with more than 255 characters raises ValueError first as this limit
+        # is removed this will not raise ValueError anymore.
+        if student is not None:
```

Comment:
@awaisdar001 are you talking about this test  **max_length** ?

I have updated the test previously its throughs ValueError when length is more than 255 character now its not .